specification/communication/data-plane/CallAutomation/preview/2023-10-03-preview/communicationservicescallautomation.json

```diff
@@ -1198,28 +1192,23 @@
         "tags": [
           "MidConnections"
         ],
-        "summary": "Stop a dialog.",
-        "description": "Stop a dialog.",
         "operationId": "CallDialog_StopDialog",
         "parameters": [
           {
             "name": "callConnectionId",
             "in": "path",
-            "description": "The call connection id",
```

Comment:
Are these removed on purpose? I think we want to use `description` everywhere and don't need `summary`.

Reply:
These changes have already been deployed previously. Not part of this code PR